scripts/tasks/sandbox-parts.ts

```diff
@@ -219,10 +219,18 @@ function addEsbuildLoaderToStories(mainConfig: ConfigFile) {
           ],
         },
         // Ensure no other loaders from the framework apply
-        ...config.module.rules.map(rule => ({
-          ...rule,
-          exclude: [/\\/template-stories\\//].concat(rule.exclude || []),
-        })),
+        ...config.module.rules.map(rule => {
+
+          // The csf loader is responsible to add metadata to CSF files (e.g. extracting JSON comments)
```

Comment:
Do we need to do something similar for Vite?

Reply:
I do not even know why we are excluding all rules per default for the sandboxes and only for Webpack. I have no context for this. We don't have to do something similar for Vite.